### ISSUE TITLE
Feature Request: Customize menubar font style

### ISSUE THREAD

**Issue by switz** [yujitach/MenuMeters#57]

**Comment by thienlh** [comment]
Yes! 

The app is great, but we need to customize Network throughput font/size, it is hard to read on my 2012 Macbook air 11 inch. I know it is possible because the same stat on iStatMenus is much more readable. 

Please take a look, thank you!

**Comment by yujitach** [comment]
Indeed. I myself want this feature. But I don't have the time myself. It's actually not very hard to implement it. @thienlh , I think it would be a great opportunity for you to learn some Objective-C and Mac programming and implement this feature. In fact I started programming on Mac because I wanted to tweak some other open-source app myself...
